frontend/app/(dashboardLayout)/equity/shares/page.tsx

```
@@ -22,20 +24,23 @@ const columns = [
   const [shareHoldings] = trpc.shareHoldings.list.useSuspenseQuery({
     companyId: company.id,
     investorId: user.roles.investor?.id ?? "",
   });
 
   const table = useTable({ data: shareHoldings, columns });
+  const currentLink = navLinks(user, company).find((link) => link.route === pathname);
 
   return (
-    <EquityLayout>
+    <>
+      {currentLink && <PageHeader currentLink={currentLink} />}
```

ESLint

[error] Potential leaked value that might cause unintentionally rendered values or rendering crashes

(react/jsx-no-leaked-render)